packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -46,40 +71,73 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
 
   static className = 'ui-accordion__title'
 
+  static slotClassNames: AccordionTitleSlotClassNames
+
   static propTypes = {
     ...commonPropTypes.createCommon(),
     active: PropTypes.bool,
-    index: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    buttonRef: customPropTypes.ref,
+    canBeCollapsed: PropTypes.bool,
+    contentId: PropTypes.string,
+    index: PropTypes.number,
     onClick: PropTypes.func,
     indicator: customPropTypes.itemShorthand,
   }
 
-  handleClick = e => {
+  static defaultProps = {
+    accessibility: accordionTitleBehavior,
+    as: 'dt',
+  }
+
+  actionHandlers: AccessibilityActionHandlers = {
+    performClick: e => {
+      e.preventDefault()
+      this.handleClick(e as any)
+    },
+  }
+
+  private handleClick = (e: React.SyntheticEvent) => {
     _.invoke(this.props, 'onClick', e, this.props)
   }
 
-  renderComponent({ ElementType, classes, unhandledProps, styles }) {
-    const { children, content, indicator, active } = this.props
+  private handleFocus = (e: React.SyntheticEvent) => {
+    e.stopPropagation()
+    _.invoke(this.props, 'onFocus', e, this.props)
+  }
+
+  renderComponent({ ElementType, classes, unhandledProps, styles, accessibility }) {
+    const { buttonRef, children, content, indicator, active } = this.props
     const indicatorWithDefaults = indicator === undefined ? {} : indicator
 
     const contentElement = (
-      <Layout
-        start={Icon.create(indicatorWithDefaults, {
-          defaultProps: {
-            name: active ? 'stardust-arrow-down' : 'stardust-arrow-end',
-            styles: styles.indicator,
-          },
-        })}
-        main={rtlTextContainer.createFor({ element: content })}
-      />
+      <Ref innerRef={buttonRef}>
+        <Box
+          onFocus={this.handleFocus}
+          onClick={this.handleClick}
+          className={AccordionTitle.slotClassNames.button}
```

Comment:
This is not button anymore, let's change the name of the slotClassNames

Reply:
One more question, do we really need the box around the Layout? The Layout itself is a div, can't we just use the Layout component here?